src/scenes/login/Login.tsx

```diff
@@ -60,15 +62,29 @@ const LoginComponent = () => {
       }
       setAlertMessage("Login was successful!");
       setAlert(true);
+      setTimeout(() => {
+        setAlert(false);
+        setLoginSuccessfull(false);
+      }, 5000);
```

Comment:
why setting timeout ?

Reply:
I removed the specific code. With that change, the timeout has reverted back to the default 3 seconds, which seems to be working well for our use case